pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -414,6 +746,35 @@ func TestInt_SecurityIntegrations(t *testing.T) {
 		assertSecurityIntegration(t, si, id, "SCIM - GENERIC", false, "")
 	})
 
+	t.Run("Show OauthPartner", func(t *testing.T) {
+		id := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		si1 := createOauthPartner(t, id, nil)
+		id2 := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		// more than one oauth partner integration is not allowed, create a custom one
+		si2 := createOauthCustom(t, id2, nil)
+
+		returnedIntegrations, err := client.SecurityIntegrations.Show(ctx, sdk.NewShowSecurityIntegrationRequest().WithLike(&sdk.Like{
+			Pattern: sdk.Pointer(id.Name()),
+		}))
+		require.NoError(t, err)
+		assert.Contains(t, returnedIntegrations, *si1)
+		assert.NotContains(t, returnedIntegrations, *si2)
+	})
+
+	t.Run("Show OauthCustom", func(t *testing.T) {
+		id := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		si1 := createOauthCustom(t, id, nil)
```

Comment:
nit-picking: in multiple tests here we create the id inside the test and pass it to the create method that return the created object (that has an id). Could we move id generation to the create* methods? It would make the tests a bit shorter.